src/components/accounts/accounts.html

```diff
@@ -1,7 +1,15 @@
-<div class="accounts">
-  <md-list-item ng-click="$ctrl.handleAddAccountClick($event)">
-    <md-icon md-font-set="material-icons"> add </md-icon>
-    <p> Add new account</p>
+
+<md-list-item ng-click="$ctrl.handleAddAccountClick($event)">
```

Comment:
you add accounts twice here
![image](https://user-images.githubusercontent.com/14272396/38331420-e9e677c6-385b-11e8-8ef3-26c7ecacf113.png)

I think it is after merging because I changed this part of code to make account component start from div